mutators.scad

```diff
@@ -64,13 +64,20 @@ module bounding_box(excess=0) {
 }
 
 
-// Module: half_of()
+// Function&Module: half_of()
 //
-// Usage:
+// Usage: as module
 //   half_of(v, [cp], [s]) ...
+// Usage: as function
+//   half_of(v, [cp], p, [s])...
 //
 // Description:
 //   Slices an object at a cut plane, and masks away everything that is on one side.
+//   * Called as a function with a path in the `p` argument, returns the
+//   intersection of path `p` and given half-space.
```

Comment:
Markdown will want this indented four more spaces as a continuation of the list item.